libraries/standard/mqtt/src/iot_mqtt_operation.c

```diff
@@ -760,7 +760,7 @@ void _IotMqtt_ProcessKeepAlive( IotTaskPool_t pTaskPool,
             swapStatus = Atomic_CompareAndSwap_u32( &( pPingreqOperation->u.operation.periodic.ping.failure ),
                                                     1,
                                                     0 );
-            IotMqtt_Assert( swapStatus == 1 );
+            IotMqtt_Assert( swapStatus == 1U );
```

Comment:
Curious why line 763 needs a `U` but lines 761 and 762 don't

Reply:
MISRA only called out the comparisons.